Nuget.config

```diff
@@ -1,10 +1,9 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
How come we remove this, is it not recommended to keep these in at the top of the file?

Reply:
Accidental - I'll revert